Bottom: b0b290209f74fac22ab4edeb9acc1cfdd6e9cc51
Top:    249d637d5f2c73789cfc1db5cdf82861f2b12641
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2014-04-17 18:43:16 -0700

Refresh of inline0

---

diff --git a/src/rsocket.c b/src/rsocket.c
index beba738..3d80690 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -1608,7 +1608,7 @@ int rconnect(int socket, const struct sockaddr *addr, socklen_t addrlen)
 	return ret;
 }
 
-static struct rs_sge *rs_get_ctrl_buf(struct rsocket *rs)
+static void *rs_get_ctrl_buf(struct rsocket *rs)
 {
 	return rs->sbuf + rs->sbuf_size +
 		RS_MAX_CTRL_MSG * (rs->ctrl_seqno & (RS_QP_CTRL_SIZE - 1));
